src/machine/behavior/is/memory/ldb.rs

```diff
@@ -1,6 +1,21 @@
 use machine::state::State;
+use machine::state::types::Octa;
+use machine::state::mem::ByteAt;
 
-pub fn ldb(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+/// load byte
+pub fn ldb(state: &mut State, x: u8, y: u8, z: u8) {
```

Comment:
It would be nice if we would have a certain structure in any of these functions denoted with comments. Example:

```rust
pub fn ldb(state: &mut State, x: u8, y:u8, z: u8) {
    // Load operands
    // <code that loads operands>

    // Execute
    // <code that executes the operand>

    // Store results
    // <code that stores the result>
}
```

Please see the example implementation for AND (src/machine/behavior/is/bitwise/and.rs) for details.